<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Table View Test</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QGroupBox" name="PlatformControlsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Platform Selector</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="simQt::EntityTreeComposite" name="EntityTreeComposite">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="DataLimitingGroupBox">
        <property name="title">
         <string>Data Limiting</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="DataLimitEnableCheckBox">
           <property name="text">
            <string>Enable Data Limiting</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="DataLimitPointsLabel">
             <property name="text">
              <string>Number of Points:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="DataLimitPointsSpinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="DataLimitTimeLabel">
             <property name="text">
              <string>Number of Seconds:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="DataLimitTimeSpinBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="DataTableGroupBox">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Data Table Controls</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="PlatformNameLabel">
          <property name="text">
           <string>Current Platform:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PlatformNameEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="DataTableListLabel">
          <property name="text">
           <string>Data Tables:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="simQt::DataTableComboBox" name="DataTableComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="TreeViewLabel">
        <property name="text">
         <string>Tree View</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="DataTableTreeView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>false</bool>
        </property>
        <attribute name="headerDefaultSectionSize">
         <number>80</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="TableViewLabel">
        <property name="text">
         <string>Table View</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableViewTest">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="AddDataTableButton">
          <property name="text">
           <string>Add Data Table</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="RemoveTableButton">
          <property name="text">
           <string>Remove Selected Table</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="TableSizeLabel">
          <property name="text">
           <string>Number of Rows in New Table:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="TableSizeSpinBox">
          <property name="maximum">
           <number>5000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="AddColumnButton">
          <property name="text">
           <string>Add Column</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="AddRowButton">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Add Row</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="RemoveRowButton">
          <property name="text">
           <string>Remove Row</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="RemoveColumnButton">
          <property name="text">
           <string>Remove Column</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="RowTimeLabel">
          <property name="text">
           <string>New Row Time:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="RowTimeSpinBox">
          <property name="maximum">
           <double>100000000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>simQt::DataTableComboBox</class>
   <extends>QWidget</extends>
   <header>simQt/DataTableComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>simQt::EntityTreeComposite</class>
   <extends>QWidget</extends>
   <header>simQt/EntityTreeComposite.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
